sotodlib/mapmaking/obs_grouping.py

```diff
@@ -123,78 +226,4 @@ def build_period_obslists(obs_info, periods, context, nset=None, wafer=None, fre
                 key = (pids[i], detset, band)
                 if key not in obslists: obslists[key] = []
                 obslists[key].append((row.obs_id, detset, band, i))
```

Comment:
The lists contain tuples `(row.obs_id, detset, band, i)`.  That's not what the function and module docstrings say.